pghoard/pghoard.py

```diff
@@ -297,6 +298,36 @@ def delete_remote_basebackup(self, site, basebackup, metadata):
                             chunk["chunk_filename"],
                         )
                     )
+        elif metadata.get("format") == BaseBackupFormat.delta_v1:
+            keep_hexdigests = set()
```

Comment:
Consider a test for the logic?

Reply:
There are tests which cover recovery functionality, they are not covering 100%, but they do cover backup and restoration, which touches pretty much every line of this code, and one test is testing the fetcher for delta data to recover. Could you tell which part exactly you want to cover?